### ISSUE TITLE
First line of text difficult to read

### ISSUE THREAD

**Issue by KnutLohse** [slub/dfg-viewer#276]
The first line at the top of the display area for the OCR text (often the heading) is difficult to read due to the blanking. A small CSS instruction (padding at the top edge) would solve this "problem".
Perhaps this correction/enhancement can be included in one of the next releases, I pointed this out years ago.
Thanks in advance
Knut

**Comment by csidirop** [comment]
You are right. That actually bothered me (and us here) too. And the solution should be as simple as you suggest. I'll see if I can prepare a PR later.

**Comment by KnutLohse** [comment]
Hi csidirop,
thanks for your quick response and action.
But: better than give the #tx-dlf-fulltextsection a top position I would prefer setting a top (and bottom?) padding. With your solution the blanking will disappear, also when scrolling - right? Furthermore I think it would be better to note the values in em (1.5em) instead of px to take into account different screen settings (i.e. larger font).
Regards, Knut

**Comment by csidirop** [comment]
Hi @KnutLohse ,

> With your solution the blanking will disappear, also when scrolling - right

No, as I have shown in my PR, the effect is kept when scrolling. We have more than enough space (60px) where the `fulltextselection` can scroll inside the `fulltext-container`.
But you are right, we could achieve the same result by increasing the top padding here (https://github.com/slub/dfg-viewer/blob/slub-master/Resources/Private/Less/modules/fulltext.less#L39) and here (https://github.com/slub/dfg-viewer/blob/slub-master/Resources/Private/Less/modules/fulltext.less#L73)

> Furthermore I think it would be better to note the values in em (1.5em) instead of px to take into account different screen settings (i.e. larger font).

Yes thats right, but I used `px` because `em` is nowhere else used. So its was more a consistency decision. 

I think @beatrycze-volk opinion is important here. If we move the textline and how.

**Comment by KnutLohse** [comment]
Hi Chris,
thank you for clarification (BTW: the links given above leading to 404).
I didn't look to the PR before writing, sorry. And I understand the idea of consistency.
Regards, Knut

**Comment by csidirop** [comment]
> Hi Chris,
> thank you for clarification (BTW: the links given above leading to 404).
> I didn't look to the PR before writing, sorry. And I understand the idea of consistency.
> Regards, Knut

I fixed the links.

Lets see what @beatrycze-volk has to say :)

**Comment by beatrycze-volk** [comment]
> > Furthermore I think it would be better to note the values in em (1.5em) instead of px to take into account different screen settings (i.e. larger font).
> 
> Yes thats right, but I used `px` because `em` is nowhere else used. So its was more a consistency decision.
> 
> I think @beatrycze-volk opinion is important here. If we move the textline and how.

Thank you @csidirop for the adjustment :)

If it goes about the question. As you mentioned, the CSS here uses mostly `px`, as it is already quite old. But moving forward we could improve it and use `em` where suitable to increase responsiveness.
